source/_integrations/pseg.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: "PSE&G Gas Meter Sensors"
+description: "Instructions on how to integrate the PSE&G gas meter sensors within Home Assistant."
+logo: pseg.png
+ha_release: 0.101
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+---
+
+[PSE&G](https://pseg.com) is an energy provider in NY and NJ, USA.
+The `pseg` sensor platform fetches your last gas consumption and cost from PSE&G.
+
+## Configuration
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: pseg
+    energize_id: YOUR_ENERGIZE_ID
+    session_id: YOUR_SESSION_ID
+```
+
+{% configuration %}
+energize_id:
+  description: The energize id corresponding to your PSE&G customer account. 
+  required: true
+  type: string
+session_id:
+  description: The session id corresponding to your PSE&G customer account. 
+  required: true
```

Comment:
According to the code base, this is optional?

<https://github.com/home-assistant/home-assistant/pull/27902/files#diff-3464f58a5d9ec8e77ebe85c9a972e414R32>

Reply:
Nice catch, @frenck. I fixed the code base to mark it as Required. Thanks!